src/utils/image_server.cljs

```diff
@@ -98,7 +98,7 @@
    "&indicatorCenterToEdge="
    (* indicator-center-to-edge status-im2.common.pixel-ratio/ratio)
    "&addRing="
-   (if ring? 1 0)
+   (if (or (not ring?) (= size 20)) 0 1)
```

Comment:
Hey there!
Could you provide more information about this? I'm trying to understand if this is here to prevent the `ring?` function from being overridden by the `override-ring?` function.

We need to come up with a better solution if that's the case.

Reply:
according to QA request we should disable ring for small size. @yakimant

Reply:
I recommend implementing this at https://github.com/status-im/status-mobile/blob/a9b63e0f22afdd9817114b09408255df54ea4bcc/src/quo2/components/avatars/user_avatar/view.cljs#L101 if that's the case